tests/test_noshow.py

```diff
@@ -26,11 +31,10 @@ def fake_postal_codes(_=None) -> pd.DataFrame:
     ).set_index("postalcode")
 
 
-def fake_model(_=None):
-    """Mock function for returning `load_model`
-
-    Needs to accept a parameter, but this parameter will be ignored
-    """
+def fake_model(model_path=None):
+    """Mock function for returning `load_model`"""
+    if model_path and "rct" in str(model_path):
```

Comment:
what if the model_path does not have rct in the name? Maybe throw an exception or something?

Reply:
If there isn't 'rct' in the model_path it should select the regular model. At least that how I see it